lib/form-mailer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+// Only process POST reqeusts.
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+  // Get the form fields and remove whitespace.
+  $email = filter_var(trim($_POST['email']), FILTER_SANITIZE_EMAIL);
+  $message = trim($_POST['message']);
+  $recipient = filter_var(trim($_POST['recipient']), FILTER_SANITIZE_EMAIL);
```

Comment:
This shouldn't be an option on the front end. With this set anyone can use this mailer can just email (spam but whatever) emails to whatever account the want. The site setting should be checked here rather than from the POST data.